backend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:alpine 
+
+RUN apk add --update git
+
+ADD . $GOPATH/src/build/
+
+WORKDIR $GOPATH/src/build/
+
+RUN go get -u "github.com/gin-gonic/contrib/cors"
+RUN go get -u "github.com/lib/pq"
+
+EXPOSE 8081
+
+CMD ["wget", "https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh"]
```

Comment:
I assume you wanted to use either RUN / ADD here and RUN for subsequent CMD lines (except the last one) since only the last CMD run will actually be executed.

As far as RUN vs ADD goes, for the first line (wgetting a file) using ADD (either for a local file or a remote one) would be more efficient since it means Docker layer can be re-used when ``docker build`` runs and container doesn't need to be rebuilt and new layer added if that file doesn't change.

Reply:
Yes, I will be changing them to RUN now.